src/main/java/org/jahia/modules/sitemap/graphql/api/GqlSitemapMutation.java

```diff
@@ -66,7 +66,7 @@ public Boolean deleteSitemapCache(
             @GraphQLName("siteKey") @GraphQLDescription("Site key") String siteKey
     ){
         try {
-            CacheUtils.refreshSitemapCache(ConversionUtils.longVal(expirationTimeDifference, 0L), siteKey);
+            CacheUtils.refreshSitemapCache(ConversionUtils.longVal(expirationTimeDifference, 4 * 60 * 60 * 1000L), siteKey);
```

Comment:
add validation that expiration value is valid (i.e. >= 0) )

Reply:
Internally it calls the isExpired so same as applied as above